jetty-core/jetty-xml/src/test/java/org/eclipse/jetty/xml/XmlConfigurationTest.java

```diff
@@ -2165,15 +2178,21 @@ public void testFooExecutableComparator()
             .sorted(EXECUTABLE_COMPARATOR)
             .map(Executable::toGenericString)
             .collect(Collectors.toList());
-        orderedMethodIds.forEach(System.out::println);
+
+        Logger slf4jLogger = LoggerFactory.getLogger(XmlConfiguration.class);
+        if (slf4jLogger.isDebugEnabled())
+        {
+            orderedMethodIds.forEach(System.out::println);
+        }
+
         String[] expectedOrder = {
-            "public void org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo()", // favor fewer args
-            "public void org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo(int)", // favor primitives over non-primitives
-            "public void org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo(java.time.temporal.Temporal)", // favor over Instant
-            "public int org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo(java.lang.String)",
+            "public void org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo()", // favour fewer args
+            "public void org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo(int)", // favour primitives over non-primitives
+            "public int org.eclipse.jetty.xml.XmlConfigurationTest$FooObj.foo(java.lang.String)", //favour classes over interfaces
```

Comment:
Can you add a `foo(CharSequence)` to the example, as `CharSequence` is an interface implemented by `String`